src/project/build.scala

```diff
@@ -146,7 +146,8 @@ object Extras {
     initialCommands in (ThisBuild, console) := "import com.microsoft.ml.spark._",
     // Use the above commands
     commands in ThisBuild ++= newCommands
-    ) ++ testOpts
+    ) ++ testOpts ++ Defaults.itSettings ++
+    addCommandAlias("adbNotebookTests", "core-test-fuzzing/it:test")
```

Comment:
I'd drop `adb` -- it's related to an implementation that can change.

Reply:
removed